### PR TITLE
ES6 Import/Export example.

### PR DESCRIPTION
**Pull request by moniika**
Refactoring main logic into 'main.js' and using ES6 import/export.

### CODE REVIEW

File: blockly-ws-search/main.js [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+  options.toolboxPosition = position;
+  const ws = Blockly.inject('blocklyDiv' + suffix, options);
+  const workspaceSearch = new WorkspaceSearch(ws);
+  workspaceSearch.init();
+  if (opt_openSearch) {
+    workspaceSearch.open();
+  }
+}
+
+document.addEventListener("DOMContentLoaded", function () { start() });
```
Comment (alschmiedt) [inline blockly-ws-search/main.js]: Add a newline here.

File: blockly-ws-search/main.js [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+  disable: true,
+  maxBlocks: Infinity,
+  oneBasedIndex: true,
+  readOnly: false,
+  //rtl: false,
+  scrollbars: true,
+  trashcan: true,
+  //toolbox: null,
+  //horizontalLayout: false,
+  //toolboxPosition: 'start',
```
Comment (alschmiedt) [inline blockly-ws-search/main.js]: We should probably remove the commented options.

**Review by alschmiedt — APPROVED** [review]

File: blockly-ws-search/main.js (hunk 2 above)
Comment (moniika) [reply blockly-ws-search/main.js]: Done

File: blockly-ws-search/main.js (hunk 1 above)
Comment (moniika) [reply blockly-ws-search/main.js]: Done.

**Pull request merged**